cookbook/solr/default.rb

```diff
@@ -0,0 +1,23 @@
+package 'java-1.8.0'
+
+execute "download solr-#{node['solr']['version']}" do
+  command "curl -o solr-#{node['solr']['version']}.tgz http://www.us.apache.org/dist/lucene/solr/#{node['solr']['version']}/solr-#{node['solr']['version']}.tgz"
+  not_if "test -d /opt/solr"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.